lychee-bin/src/commands/check.rs

```diff
@@ -51,9 +51,11 @@ where
     let pb = if params.cfg.no_progress {
         None
     } else {
-        let bar = ProgressBar::new_spinner().with_style(ProgressStyle::default_bar().template(
-            "{spinner:.red.bright} {pos}/{len:.dim} [{elapsed_precise}] {bar:25} {wide_msg}",
-        ).expect("Valid progress bar"));
+        let bar = ProgressBar::new_spinner().with_style(
+            ProgressStyle::default_bar()
+                .template("{spinner:.197.bright} {pos}/{len:.dim} ETA {eta} {bar:.dim} {wide_msg}")
```

Comment:
can you replace default_bar() and template() by with_template()?

Reply:
Good idea, thanks.